SampleLibrary/iOS/BasicExternalObject.swift

```diff
@@ -164,6 +186,16 @@ public class BasicExternalObjectOffline: GXExternalObjectBase {
 		}
 	}
 	
+	@objc(addNumbers::)
+	public class func addNumbers(_ fisrtAddend: Int,_ secondAddend: Int) -> Int {
+		return fisrtAddend + secondAddend
+	}
+	
+	@objc(printMessage:)
+	public class func printMessage(_ message: String) {
+		GXActionExObjEventsHelper.dispatchExternalObjectEvent("BasicExternalObject.OnMessagePrinted", withParameters: [message])
```

Comment:
Use `GXExternalObjectBase` helper method for firing EO events from offline:
- `dispatchExteralObjectEvent(_:withParameters:)`: internally combines the given event name with `externalObjectName` and post notification on the current thread.
- `fireExternalObjectEvent(_: parameters:)`: internally combines the given event name with `externalObjectName` and post notification on the main thread and converts parameters if needed.
```suggestion
		dispatchExteralObjectEvent("OnMessagePrinted", withParameters: [message])
```